src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -95,6 +95,7 @@
     "421611": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "421613": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "11155111": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
+    "245022926": "0xAb5209Fbd1512008836259Cdb317199ED57faA83",
```

Comment:
This address is not the deterministic address

Reply:
hey @moisses89 , I deployed the safe-contracts again using 0x914d7fec6aac8cd542e72bca78b30650d45643d7 as the singleton factory deployed by @Uxio0 on Neon. Please review again.

Reply:
@Sj-001 which tag branch did you use on safe-contracts? you should use [v1.3.0-libs.0](https://github.com/safe-global/safe-contracts/tree/v1.3.0-libs.0) as is described here https://github.com/safe-global/safe-contracts#deploy 
I'm asking because are not still the deterministic addresses.

Reply:
Gotcha, deployed again, please review, I think its the correct one now:) @moisses89